### PR TITLE
Action returns if release is newest

### PR DESCRIPTION
**Pull request by rashmipriya12**
**Context:**
[US150075](https://rally1.rallydev.com/#/?detail=/userstory/692195977405&fdp=true): Update deploy to dev CI to include run smoke tests and switch prerelease to release

returns true for newest release

**Quality:** 🤞

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ jobs:
       isPush: ${{ steps.workflowEnv.outputs.isPush }}
       isTagged: ${{ steps.workflowEnv.outputs.isTagged }}
       isRelease: ${{ steps.workflowEnv.outputs.isRelease }}
+      isNewest: ${{ steps.workflowEnv.outputs.isNewest }}
```
Comment (AntonBazhal) [inline README.md]: I'd use `isLatest` terminology as it checks whether the CI is running the latest code

File: action.yml [hunk 2]
```diff
@@ -25,9 +25,12 @@ outputs:
   isRelease:
     description: 'true if workflow is running against a release'
     value: ${{ steps.parse-workflow-env.outputs.isRelease }}
+  isNewest:
+    description: 'true if workflow is running against the newest release'
```
Comment (AntonBazhal) [inline action.yml]:
```suggestion
    description: 'true if workflow is running the latest code'
```

File: parse-workflow-env.sh [hunk 3]
```diff
@@ -24,12 +25,19 @@ if [[ $GITHUB_EVENT_NAME == "schedule" ]]; then
   IS_PUSH='false'
 fi
 
+current=$((git show-ref --hash --verify HEAD))
```
Comment (AntonBazhal) [inline parse-workflow-env.sh:28]: I hope git cli is available in our runners

File: README.md (hunk 1 above)
Comment (rashmipriya12) [reply README.md]: so I thought about using `isLatest` but as per github, the tag with `latest` is the latest release and not pre-released release created after `latest` release. I can change it to `isLatest` I am already annoyed with it.

File: parse-workflow-env.sh (hunk 3 above)
Comment (rashmipriya12) [reply parse-workflow-env.sh:28]: We wasted time in getting this brand commit, github action has variable GITHUB_SHA for it, but looks like this should work as per [stackoverflow](https://stackoverflow.com/questions/58886293/getting-current-branch-and-commit-hash-in-github-action). 
After this is merged, I will test it in the other repo I have created before using it in actual workflow.

Comment (AntonBazhal) [reply parse-workflow-env.sh:28]: Yep, I forgot about `GITHUB_SHA`. Both approaches should work, so up to you. We still need to use git cli to get main branch sha though.

File: README.md (hunk 1 above)
Comment (AntonBazhal) [reply README.md]: Haha, my head exploded from so many occurrences of `latest` in one sentence. I think there is a misunderstanding of what we are trying to surface. This action can be run on any branch, not only on a tag. So the goal is to check that the code is the latest (aka is the same as in the default branch).

Comment (rashmipriya12) [reply README.md]: well that make sense now :) updated the variable to `isLatest` everywhere.

**Review by AntonBazhal — APPROVED** [review]
🌶️

**Pull request merged**